src/odemis/acq/stream/_sync.py

```diff
@@ -37,6 +37,7 @@
 from odemis.acq import leech
 from odemis.acq.leech import AnchorDriftCorrector
 from odemis.acq.stream._live import LiveStream
+from odemis.driver import ephemeron
```

Comment:
The acq module should know nothing about the drivers. If there are constants to care about, you can place them in odemis.model . That said, I don't think it's necessary to check the `scanState` anyway.